.bem/make.js

```diff
@@ -106,8 +106,9 @@ MAKE.decl('SpecNode', {
 
     getLevels : function() {
         return [
-            'libs/bem-core/common.blocks',
-            'common.blocks'
+            'spec.blocks',
+            'common.blocks',
+            'libs/bem-core/common.blocks'
```

Comment:
I got an error if removing it:
```
16:25:28.697 - info: [t] Build time for "common.specs/ecma__array/spec-js/spec-js.bemhtml" [2]: 719ms
TypeError: First argument needs to be a number, array or string.
    at new Buffer (buffer.js:188:15)
    at Writer.self.write (/Users/qfox/free/es5-shims/node_modules/bem/node_modules/q-io/writer.js:48:23)
    at /Users/qfox/free/es5-shims/node_modules/bem/node_modules/q-io/fs-common.js:84:34
    at _fulfilled (/Users/qfox/free/es5-shims/node_modules/bem/node_modules/q/q.js:798:54)
    at self.promiseDispatch.done (/Users/qfox/free/es5-shims/node_modules/bem/node_modules/q/q.js:827:30)
    at Promise.promise.promiseDispatch (/Users/qfox/free/es5-shims/node_modules/bem/node_modules/q/q.js:760:13)
    at /Users/qfox/free/es5-shims/node_modules/bem/node_modules/q/q.js:526:49
    at flush (/Users/qfox/free/es5-shims/node_modules/bem/node_modules/q/q.js:108:17)
    at process._tickCallback (node.js:448:13)
```

No, I don't know how to remove it. I tried already ;-)